### PR TITLE
Copy samples. Test configs. Resolves #1 #2

### LINKED ISSUE #1: Add starter d3 files basic html js css
D3 sample

### LINKED ISSUE #2: Add d3 typings
Intellisense for 99999x dev vs 0.1x dev

### PR DESCRIPTION
**Pull request by slimlime**
Should solve some of the team convention/structure set up.
Resolves #1 
Resolves #2 

---

Check out this force rewriting history rebase pull push

Added in the sketchy clone from completely unrelated separate local repo from laptop last night copy.

Hmmm force-direct graphs
Hmm mind mapping clustering on top of geo maps.

Should update package json

Hope my gpgsign expiry renewal worked.
Verified tag is 👌

### CODE REVIEW

**Comment by slimlime** [comment]
O should use tool to copy the nice emoji labels from previous repos

**Comment by slimlime** [comment]
Commitizen interesting multi-step (_slow_) interactive CLI
call using `cz` and watch it conventionalise
todo commitlint

will add in the bulk instructions (npm i global save dev etc) on review later

let me know when the novelty fades and still feels like an onerous process significantly detrimental to our developer flow

**Comment by slimlime** [comment]
bad bot!

**Comment by slimlime** [comment]
interesting check
> 

> ✅ Skipping update for ':package: dependecies' label. Same color and description
> ✅ Skipping update for 'duplicate' label. Same color and description
> 🔨 Updating 'enhancement' label with color 'a22eef'
> ✨ Renaming 'wontfix' label to 'wontfix_it' and set color '000000' and desc 'This will not be worked on'
> Error: Cannot rename "wontfix" label: Not Found
> 🚫️ Excluding 'good first issue' from deletion.
> 🚫️ Excluding 'help wanted' from deletion.
> Error: Errors have occurred. Please check generated annotations.


Not sure what that derived label thing was in template

Probs in doco

@seant1 Would you like to look into it or see serve the index.html to get same mind map demo prototype before accepting the pull request merge?


Some more set up to do after

**Comment by slimlime** [comment]
or gotta-go-fast codepen d3.js 👀  dev

codespaces

File: .github/labels.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Labels file for actions/github-labeler, crazy-max/ghaction-github-labeler ...
```
Comment (slimlime) [inline .github/labels.yml:1]: the templates file

github labels as code

**Comment by seant1** [comment]
> @seant1 Would you like to look into it or see serve the index.html to get same mind map demo prototype before accepting the pull request merge?

Yeah I'll look into it. Gotta do some d3 learning firsts

**Review by slimlime — COMMENTED** [review]
> > @seant1 Would you like to look into it or see serve the index.html to get same mind map demo prototype before accepting the pull request merge?
> 
> Yeah I'll look into it. Gotta do some d3 learning firsts

We can rewrite it better - Added the sample purely to test preview/prototyping seeing that it works in the first place
Or make separate sandbox branch to replicate some of the doco examples e.g. tree with four nodes we can write more simply
:D
per component file if doing multiple demos
maybe something like this for learnings https://avin.github.io/d3-sandbox/

**Comment by slimlime** [comment]
sandboxiii @seant1 
Screen Shot 2020-12-06 at 9 57 53 am

**Review by seant1 — APPROVED** [review]

**Pull request merged**